cli/user/src/test/java/io/pravega/cli/user/scope/ScopeCommandsTest.java

```diff
@@ -52,7 +52,7 @@ public static void shutDown() {
         }
     }
 
-    @Test(timeout = 5000)
+    @Test(timeout = 10000)
```

Comment:
This is not related, but these two tests were flaking due to timeout. This appears to fix it.